src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -466,6 +466,41 @@ public void MaxRequestContentBufferSize_Set_ThrowsPlatformNotSupportedException(
         }
 
         [Fact]
+        public async Task GetAsync_ResponseWithSetCookieHeaders_AllCookiesRead()
+        {
+            // Cookie names/values, with one of the values deliberately over 128 characters,
+            // which is the size of WinHttpHandler's initial response header buffer size, to
```

Comment:
>which is the size of WinHttpHandler's initial response header buffer size

Is there any buffer size issues that CurlHandler has? Will this test also cover any interesting edge cases for CurlHandler?